app/views/accounts/index.html.erb

```diff
@@ -127,5 +127,10 @@
         <% end %>
       </tbody>
     </table>
+
+    <div class="pagination-wrapper">
+      <%= paginate @users, theme: 'twitter-bootstrap-4' %>
```

Comment:
i have an empty wrapper here. are we using the following defaults?

```
default_per_page      # 25 by default
max_per_page          # nil by default
max_pages             # nil by default
window                # 4 by default
outer_window          # 0 by default
left                  # 0 by default
right                 # 0 by default
page_method_name      # :page by default
param_name            # :page by default
params_on_first_page  # false by default
```

https://github.com/kaminari/kaminari#general-configuration-options

Reply:
The wrapper is empty when there are not enough records for pagination.
The wrapper could also just be omitted.

Reply:
> I overwrote none of the defaults.

ok, my 24 accounts match the `25` per page default, so that's why i do not see a pagination.

> The wrapper could also just be omitted.

ok please do so. i have also checked quickly, that a removal does not affect anything. that way we do not render that empty dom elements.